Makefile

```diff
@@ -78,7 +78,7 @@ calico_containers/calico-node.tar: caliconode.created
 	docker save --output calico_containers/calico-node.tar calico/node
 
 st: binary calico_containers/busybox.tar calico_containers/routereflector.tar calico_containers/calico-node.tar run-etcd
-	sudo dist/calicoctl checksystem --fix
+	sudo dist/calicoctl checksystem
```

Comment:
I think we should just remove this call. It's not needed to run every time we run the tests. Developers system and CI systems will either work or not and it won't change. If it's not working, then they can run the command manually.